forgot-password.html

```diff
@@ -26,7 +26,7 @@ <h1>Forgot Your Password?</h1>
                     </div>
                 </form>
                 <div>
-                   <span class="body-text">Enter your username or email address here<br>and we will send you instructions<br>on how to create a new password.</span>
+                   <span class="body-text">Enter your username or email address here and we will send you instructions on how to create a new password.</span>
```

Comment:
I don't think we need these breaks anymore. The text seems to be responsive and break appropriately as the page is resized.